multiset.cabal

```diff
@@ -25,7 +25,7 @@ Library
   include-dirs:       include
   default-extensions: CPP
   ghc-options:        -Wall
-  build-depends:      containers >= 0.5, base >= 4 && < 5, deepseq >=1.2 && <1.5
+  build-depends:      containers >= 0.5.4, base >= 4 && < 5, deepseq >=1.2 && <1.5
```

Comment:
Does that new bound seem acceptable? v0.5.4 was released in 2014.

Reply:
The bound on base probably also needs to be updated, since your changes seem to require Semigroup => Monoid.

Reply:
> Alternatively I can add a bit of CPP.

I ended up doing that. This patch builds with GHC-7.8 now.